Models/AuthenticationModels/ForgotPasswordRequest.cs

```diff
@@ -4,6 +4,6 @@ namespace LethalCompany_Backend.Models.AuthenticationModels;
 
 public class ForgotPasswordRequest
 {
-    [MaxLength(64)]
+    [MaxLength(64), Required]
```

Comment:
required unnecessary, Entity framework sees that the field is not nullable so it is already required.